plugins/ibm_qradar/help.md

```diff
@@ -0,0 +1,1068 @@
+# Description
+
+IBM QRadar reduces billions of events and flows into a manageable number of actionable offenses that are prioritized by their impact on your business operations. This plugin allows you to use IBM QRadar to orchestrate and automate Ariel search queries and automate offense management
+
+# Key Features
+
+Identify key features of plugin.
+
+# Requirements
+
+
+# Supported Product Versions
+
+* 7.3.3
+
+# Documentation
+
+## Setup
+
+The connection configuration accepts the following parameters:
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|hostname|string|None|True|Hostname for the QRadar application|None|https://example.com|
+|password|password|None|True|QRadar Password|None|password|
+|username|string|None|True|QRadar Username|None|username|
+
+Example input:
+
+```
+{
+  "hostname": "127.0.0.1",
+  "password": "password",
+  "username": "username"
```

Comment:
We use the username as "user1" as standard for username. https://docs.rapid7.com/insightconnect/style-guide/